### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by luisFilipePT**
Minor improvements

Improved Favion
image

Twitter links improve placement
image

Fix og-image
image

Remove dead code

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,239 @@
 
-public/
 .cache/
-node_modules/
\ No newline at end of file
+
```
Comment (coderbyheart) [inline .gitignore:3]: Do not revert previous changes.

File: gatsby-config.js [hunk 2]
```diff
@@ -72,6 +72,7 @@ module.exports = {
         theme_color: `#f5f0eb`,
         start_url: `/`,
         display: `standalone`,
+        icon: require.resolve('./src/images/favicon.png'),
```
Comment (coderbyheart) [inline gatsby-config.js]: I removed the house favicon, in favour of a simple SVG icon: 🏠
So, please don't re-add it.

**Review by coderbyheart — CHANGES_REQUESTED** [review]
Updates push out menu toggle on mobile off screen.

File: .gitignore (hunk 1 above)
Comment (luisFilipePT) [reply .gitignore:3]: Well I didn't revert them, I guess, just reorganize them:

Line 82 (node_modules)
```
# Dependency directories
node_modules/
```

Line 126 (public)
```
# Uncomment the public line if your project uses Gatsby
# https://nextjs.org/blog/next-9-1#public-directory-support
# https://create-react-app.dev/docs/using-the-public-folder/#docsNav
public
```

But if you prefer I can move them to the top again 😄

File: gatsby-config.js (hunk 2 above)
Comment (luisFilipePT) [reply gatsby-config.js]: ok, will remove the entry in the config and the image file but I don't see the 🏠 being used in the live version

image

Maybe redeploy with cache clear on netlify 🤔

Comment (luisFilipePT) [reply gatsby-config.js]: Done https://github.com/homeferences/website/pull/10/commits/f548d61a5706d6b049349bee0c196f149f81b208

**Comment by luisFilipePT** [comment]
> Updates push out menu toggle on mobile off screen.

can you share a screenshot?

**Comment by luisFilipePT** [comment]
> > Updates push out menu toggle on mobile off screen.
> 
> can you share a screenshot?

Ok, I see what you mean, will fix it tomorrow, thanks

**Comment by luisFilipePT** [comment]
> Updates push out menu toggle on mobile off screen.

I think its good now

File: gatsby-config.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ module.exports = {
       'Homeferences are Conferences which happen virtually, so you can attend from your home.',
     url: 'https://homeferences.netlify.com/',
     author: author,
-    image: `${__dirname}/src/images/homeferences.jpeg`,
+    image: `https://homeferences.netlify.com/og-image.jpeg`,
```
Comment (coderbyheart) [inline gatsby-config.js:17]: Don't use hardcoded hostnames.

File: .gitignore (hunk 1 above)
Comment (coderbyheart) [reply .gitignore:3]: No, please revert this file to this commit: 1a8b0ba178fd9112b719e8cda12ea0e11897e9a3

File: src/images/README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+No images used now but keep directory for future use
```
Comment (coderbyheart) [inline src/images/README.md:1]: Remove. If someone needs this directory, they can add it again.

**Review by coderbyheart — CHANGES_REQUESTED** [review]

**Comment by coderbyheart** [comment]
Thanks!

**Pull request closed without merging**